static/js/rules/PropertySelect.js

```diff
@@ -53,7 +53,10 @@ PropertySelect.prototype.addOption = function(name, ruleFragment, selected) {
     elt.classList.add('selected');
   }
   elt.dataset.ruleFragment = JSON.stringify(ruleFragment);
-  elt.textContent = name;
+  const nameElt = document.createElement('span');
+  nameElt.classList.add('property-select-name');
+  nameElt.textContent = name;
```

Comment:
If you set the `title` attribute here, we'll get some hover text. At least then you can differentiate the properties.

Reply:
Oh, nice idea! I'll add it